ai-data/generative-apis/reference-content/supported-models.mdx

```diff
@@ -0,0 +1,49 @@
+---
+meta:
+  title: Supported models
+  description: This page lists which open-source chat or embedding models Scaleway is currently hosting.
+content:
+  h1: Supported models
+  paragraph: Generative APIs have serverless AI models hosted at Scaleway: no need to configure hardware or deploy your own models.
+tags: generative-apis ai-data supported-models
+dates:
+  validation: 2024-09-02
+  posted: 2024-09-02
+---
+
+<Message type="important">
+  This service is free while in beta. [Specific terms and conditions](https://www.scaleway.com/en/contracts/) apply.
+</Message>
+
+Our [Chat API](/ai-data/generative-apis/how-to/query-text-models) has built-in support for the most popular instruct models.
+
+## Chat models
+
+| Provider | Model string | Context window | License | Model card |
+|-----------------|-----------------|-----------------|-----------------|-----------------|
+| Meta        | `llama-3.1-8b-instruct`  | 128k  | [Llama 3.1 Community License Agreement](https://llama.meta.com/llama3_1/license/) | [HF](https://huggingface.co/meta-llama/Meta-Llama-3.1-8B-Instruct) |
+| Mistral      | `mistral-nemo-instruct-2407`                 | 128k | [Apache-2.0](https://www.apache.org/licenses/LICENSE-2.0) | [HF](https://huggingface.co/mistralai/Mistral-Nemo-Instruct-2407) |
+
+<Message type="tip">
+  If you're not sure which chat model to use, we currently recommend Llama 3.1 8B Instruct (`llama-3.1-8b-instruct`) to get started.
+</Message>
+
+<Message type="note">
+  Missing a model? [Tell us what you would like to see here](https://feature-request.scaleway.com/?tags=ai-services), and we might add it shortly.
+</Message>
+
+## Embedding models
+
+Our [Embeddings API](/ai-data/generative-apis/how-to/query-embedding-models) provides built-in support for the following models, hosted in Scaleway data centers, available via serverless endpoints.
+
+| Provider | Model string | Model size | Embedding dimension | Context window |  License | Model card |
+|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|-----------------|
+| SBERT        | `sentence-t5-xxl`  | 5B  | 768 | 512 | [Apache-2.0](https://www.apache.org/licenses/LICENSE-2.0) | [HF](https://huggingface.co/sentence-transformers/sentence-t5-xxl) |
+
+## Request a model
+
+**Don't see a model you want to use?** [Tell us or vote for what you would like to add here.](https://feature-request.scaleway.com/?tags=ai-services)
```

Comment:
Redundant with line 31?

Reply:
Yes on purpose, wanted to insist and collect feedback